src/main/java/spring/boot/com/sbparser/Review.java

```diff
@@ -0,0 +1,156 @@
+    private int helpfulnessDenominator;
+    private int score;
+    private LocalDateTime time;
+    private String summary;
+    private String text;
+
+    public int getReviewId() {
+        return reviewId;
+    }
+
+    public void setReviewId(int reviewId) {
+        this.reviewId = reviewId;
+    }
+
+    public String getProductId() {
+        return productId;
+    }
+
+    public void setProductId(String productId) {
+        this.productId = productId;
+    }
+
+    public String getUserId() {
+        return userId;
+    }
+
+    public void setUserId(String userId) {
+        this.userId = userId;
+    }
+
+    public String getProfileName() {
+        return profileName;
+    }
+
+    public void setProfileName(String profileName) {
+        this.profileName = profileName;
+    }
+
+    public int getHelpfulnessNumerator() {
+        return helpfulnessNumerator;
+    }
+
+    public void setHelpfulnessNumerator(int helpfulnessNumerator) {
+        this.helpfulnessNumerator = helpfulnessNumerator;
+    }
+
+    public int getHelpfulnessDenominator() {
+        return helpfulnessDenominator;
+    }
+
+    public void setHelpfulnessDenominator(int helpfulnessDenominator) {
+        this.helpfulnessDenominator = helpfulnessDenominator;
+    }
+
+    public int getScore() {
+        return score;
+    }
+
+    public void setScore(int score) {
+        this.score = score;
+    }
+
+    public LocalDateTime getTime() {
+        return time;
+    }
+
+    public void setTime(LocalDateTime time) {
+        this.time = time;
+    }
+
+    public String getSummary() {
+        return summary;
+    }
+
+    public void setSummary(String summary) {
+        this.summary = summary;
+    }
+
+    public String getText() {
+        return text;
+    }
+
+    public void setText(String text) {
+        this.text = text;
+    }
+
+    @Override
+    public String toString() {
+        return "Review{" + "reviewId=" + reviewId + ", productId='" + productId + '\''
+                + ", userId='" + userId + '\'' + ", profileName='" + profileName + '\''
+                + ", helpfulnessNumerator=" + helpfulnessNumerator
+                + ", helpfulnessDenominator=" + helpfulnessDenominator
+                + ", score=" + score + ", time=" + time + ", summary='" + summary + '\''
+                + ", text='" + text + '\'' + '}';
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+
+        Review review = (Review) o;
+
+        if (reviewId != review.reviewId || helpfulnessNumerator != review.helpfulnessNumerator
+                || helpfulnessDenominator != review.helpfulnessDenominator
+                || score != review.score) {
+            return false;
+        }
+        if (productId != null ? !productId.equals(review.productId) : review.productId != null) {
```

Comment:
let's use Object.equals() here in these comparisons

Reply:
done